tests/ert/unit_tests/ensemble_evaluator/test_ensemble_evaluator.py

```diff
@@ -469,11 +471,18 @@ async def test_ensure_multi_level_events_in_order(evaluator_to_use):
         # about realizations, the state of the ensemble up until that point
         # should be not final (i.e. not cancelled, stopped, failed).
         ensemble_state = snapshot_event.snapshot.get("status")
+        final_event_was_EETerminated = False
```

Comment:
Can we also have another flag for testing that we get both a snapshot/snapshotupdate in addition to the terminated event?

Reply:
Done!